### PR TITLE
Use the new yaml for Android X

### PR DESCRIPTION
**Pull request by mattleibow**
This PR needs a few others to be merged first:
 - https://github.com/xamarin/XamarinComponents/pull/649 for the templating features
 - https://github.com/xamarin/XamarinComponents/pull/652 for the Guava dependency

### CODE REVIEW

File: source/AndroidXProject.cshtml [hunk 1]
```diff
@@ -151,7 +151,7 @@
     <!-- PackageReference -->
     @foreach (var dep in @Model.NuGetDependencies) {
       if (!dep.IsProjectReference) {
-      <PackageReference Include="@(dep.NuGetPackageId)" Version="@(dep.NuGetVersion)" />
+      <PackageReference Include="@(dep.NuGetPackageId)" Version="@(dep.NuGetVersion)" PrivateAssets="none" />
```
Comment (mattleibow) [inline source/AndroidXProject.cshtml]: This is to make sure that the .nupkg will correctly import the dependencies - such as Guava.

File: source/androidx.slice/slice-view/transforms/Metadata.xml [hunk 2]
```diff
@@ -2,5 +2,5 @@
     <attr path="/api/package[@name='androidx.slice.widget']/class[@name='LargeSliceAdapter']/method[@name='onCreateViewHolder' and count(parameter)=2 and parameter[1][@type='android.view.ViewGroup'] and parameter[2][@type='int']]" name="return">androidx.recyclerview.widget.RecyclerView.ViewHolder</attr>
     <attr path="/api/package[@name='androidx.slice.widget']/class[@name='LargeSliceAdapter']/method[@name='onBindViewHolder' and count(parameter)=2 and parameter[1][@type='androidx.slice.widget.LargeSliceAdapter.SliceViewHolder'] and parameter[2][@type='int']]/parameter[1]" name="type">androidx.recyclerview.widget.RecyclerView.ViewHolder</attr>
     <attr path="/api/package[@name='androidx.slice.widget']/class[@name='SliceView']/method[@name='onChanged' and count(parameter)=1 and parameter[1][@type='androidx.slice.Slice']]/parameter[1]" name="managedType">Java.Lang.Object</attr>
-    <attr path="/api/package[@name='androidx.slice.widget']/interface[@name='SliceActionView.SliceActionLoadingListener']" name="visibility">public</attr>
+    <remove-node path="/api/package[@name='androidx.slice.widget']/interface[@name='SliceActionView.SliceActionLoadingListener']" />
```
Comment (Redth) [inline source/androidx.slice/slice-view/transforms/Metadata.xml]: Just want to double check this change?

**Pull request merged**